GEOSagcm_GridComp/GEOSphysics_GridComp/GEOSsurface_GridComp/GEOSland_GridComp/GEOScatch_GridComp/catchment.F90

```diff
@@ -1865,10 +1864,17 @@ SUBROUTINE energy1 (                                                     &
                   )
 
 
+      ! MB: FOXY only called here, ZBAR unchanged
+      CALL OXYFAC (                                                            &
+                   NCH, ZBAR, POROS,                                           &
```

Comment:
ZBAR is input to this function but not calculated yet, see line above that you missed to add.

Reply:
Cannot add it to the line above. So I write it here. Add at line 1842:
      !ZBAR defined here positive below ground and in meter
      ZBAR(CHNO)=SQRT(1.e-20+CATDEF(CHNO)/BF1(CHNO))-BF2(CHNO)

compare with 3184 of https://github.com/mbechtold/PEATCLSM_T/blob/main/catchment.F90